aws/resource_aws_ec2_client_vpn_endpoint_test.go

```diff
@@ -73,26 +73,61 @@ func testSweepEc2ClientVpnEndpoints(region string) error {
 	return nil
 }
 
-func TestAccAwsEc2ClientVpnEndpoint_basic(t *testing.T) {
+func TestAccAwsEc2ClientVpn(t *testing.T) {
```

Comment:
For other readers, this pattern is not likely to perpetuate further than this experimental implementation of semaphore handling.